packages/twenty-front/src/modules/settings/data-model/object-details/components/SettingsObjectItemTableRow.tsx

```diff
@@ -46,7 +46,9 @@ export const SettingsObjectItemTableRow = ({
   return (
     <StyledObjectTableRow key={objectItem.namePlural} onClick={onClick}>
       <StyledNameTableCell>
-        {!!Icon && <Icon size={theme.icon.size.md} />}
+        {!!Icon && (
+          <Icon size={theme.icon.size.md} color={theme.font.color.primary} />
```

Comment:
Here you set the icon's `color` prop, but #3103 refers to the icon's stroke width, so it matches the text's font-weight. Instead of the color, the `stroke` prop should be set with `theme.icon.stroke.sm`.

Also, can you apply the same change to the Icon's stroke width in this other component: `SettingsObjectFieldItemTableRow`?

Reply:
@thaisguigon  thanks for correcting me, My bad. will make the require changes

Reply:
@thaisguigon I have update the Icon stroke in both the files please review it once.